<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDBM文件系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>开始</string>
    </property>
    <widget class="QMenu" name="menuRemove">
     <property name="title">
      <string>移除设备</string>
     </property>
    </widget>
    <addaction name="insertDevAct"/>
    <addaction name="menuRemove"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="importFileAct"/>
    <addaction name="parmSetAct"/>
    <addaction name="formatDevAct"/>
   </widget>
   <widget class="QMenu" name="menuStatus">
    <property name="title">
     <string>当前</string>
    </property>
    <addaction name="detailStatusAct"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuTool"/>
   <addaction name="menuStatus"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="parmSetAct">
   <property name="text">
    <string>格式化设置</string>
   </property>
   <property name="statusTip">
    <string>工具</string>
   </property>
  </action>
  <action name="formatDevAct">
   <property name="text">
    <string>格式化</string>
   </property>
  </action>
  <action name="insertDevAct">
   <property name="text">
    <string>插入设备</string>
   </property>
   <property name="statusTip">
    <string>开始 </string>
   </property>
  </action>
  <action name="importFileAct">
   <property name="text">
    <string>文件导入</string>
   </property>
  </action>
  <action name="detailStatusAct">
   <property name="text">
    <string>设备状态</string>
   </property>
   <property name="statusTip">
    <string>当前</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
